src/alerts.js

```diff
@@ -0,0 +1,42 @@
+const client = require("./api_client.js");
+const alertsUrl =
+  "https://api-v3.mbta.com/alerts?fields%5Balert%5D=updated_at%2Ceffect%2Cdescription%2Cheader%2Cinformed_entity&filter%5Bactivity%5D=ALL";
+
+function emptyStringIfNull(input) {
+  return input ? input : "";
+}
+
+exports.get = () => {
+  return client.get(alertsUrl).then((response) => {
+    return {
+      id: "alerts",
+      alerts: response.data
+        .filter((alert) => {
+          return (
+            alert.attributes.effect === "ELEVATOR_CLOSURE" ||
+            alert.attributes.effect === "ESCALATOR_CLOSURE"
+            //     || alert.attributes.effect === "ACCESS_ISSUE"
```

Comment:
we typically don't include commented out code in commits -- I think this is something that could easily be added in later if necessary

Reply:
agreed. i was leaving it in as i wasn't sure if we wanted this or not. after talking with katie i think it should be deleted